### PR TITLE
Bug 1904155: Make utilization chart domain match duration

### PR DESCRIPTION
**Pull request by TheRealJon**
Update all utilization cards so that the chart x domains will match the currently selected duration.

### Before
screencast-localhost_9000-2021 06 01-10_27_19

### After
screencast-localhost_9000-2021 06 01-10_09_08

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/utilization-card.tsx [hunk 1]
```diff
@@ -3,42 +3,36 @@ import { useTranslation } from 'react-i18next';
...
           <PrometheusUtilizationItem
             title={t('ceph-storage-plugin~Used Capacity')}
             utilizationQuery={INDEPENDENT_UTILIZATION_QUERIES[StorageDashboardQuery.USED_CAPACITY]}
             duration={duration}
             humanizeValue={humanizeBinaryBytes}
             byteDataType={ByteDataTypes.BinaryBytes}
-            setTimestamps={setTimestamps}
+            startDate={startDate}
+            endDate={endDate}
+            updateEndDate={updateEndDate}
```
[the remark this answers is not in the document]
Comment (TheRealJon) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/utilization-card.tsx:35]: Yeah, probably a good use case for redux here.

[the remark this answers is not in the document]
Comment (TheRealJon) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/utilization-card.tsx]: @spadgett I've updated to use redux state

**Review by spadgett — APPROVED** [review]
/lgtm

Much cleaner 👍

**Comment by TheRealJon** [comment]
@spadgett, I rebased this, could you tag again?

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
Looks like the kubevirt tests are broken, but I don't think it's related to this PR:

> Error from server (InternalError): error when creating "https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/main/deploy/webhooks.yaml": Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": x509: certificate signed by unknown authority

**Comment by TheRealJon** [comment]
@spadgett  rebased again, needs another lgtm tag.

**Comment by spadgett** [comment]
kubevirt tests are fixed by #9430

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**